### DISCUSSION TITLE
FormBuilderChoiceChip check color

### DISCUSSION THREAD

**Discussion by Alsahim0420** [danvick/flutter_form_builder#1393] [General]
I have not been able to change the color of the check or disable it in the latest version 9.2.1

**Comment by deandreamatias** [comment]
Hi @Alsahim0420 
Now had the property `showCheckmark` on `FormBuilderChoiceChip`, added on https://github.com/flutter-form-builder-ecosystem/flutter_form_builder/issues/1317 
About color, I created an issue to add `checkmarkColor` and others: https://github.com/flutter-form-builder-ecosystem/flutter_form_builder/issues/1448
